src/commons/api/admin.rs

```diff
@@ -479,6 +508,7 @@ impl Eq for TaCertDetails {}
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 #[allow(clippy::large_enum_variant)]
 #[serde(rename_all = "snake_case")]
+#[serde(tag = "type")]
```

Comment:
As ParentCaContact is used by `daemon/http/server.rs` does this structural change to the way this type is (de)serialized to/from JSON also impact the Krill HTTP API and does the OpenAPI definition thereofre need updating to match?